packages/button/components/ButtonBase.tsx

```diff
@@ -37,6 +37,10 @@ export interface ButtonProps extends SharedLinkProps {
    * if the button triggers new content to appear (e.g.: modals and dropdowns)
    */
   ariaHaspopup?: boolean;
+  /**
+   * should be used if the button does not contain text children. For example: a button that is just an icon
+   */
+  ariaLabel?: string;
```

Comment:
it looks like i can provide whatever aria value if i use kebap-case - like `aria-label="test"`.

that's mainly an observation. i also have mixed feelings about finding that prop in the interface while not having an explicit usage.